src/btree/bt_handle.c

```diff
@@ -115,8 +115,11 @@ __wt_btree_open(WT_SESSION_IMPL *session, const char *op_cfg[])
     if (!WT_PREFIX_SKIP(filename, "file:"))
         WT_ERR_MSG(session, EINVAL, "expected a 'file:' URI");
 
-    WT_ERR(__wt_block_manager_open(session, filename, dhandle->cfg, forced_salvage,
-      F_ISSET(btree, WT_BTREE_READONLY), btree->allocsize, &btree->bm));
+    WT_WITH_BUCKET_STORAGE(btree->bstorage, session,
```

Comment:
I thought I removed that in the merge from WT-7428. The `bstorage` is now in `tiered->bstorage`. Yikes! I missed removing it from `include/btree.h`. Please do that here. There should not be a `btree->bstorage`. I removed the code setting it in `bt_handle.c` but neglected to remove the field.

Reply:
@sueloverso Why don't we keep the `btree->storage` and populate it with the bucket_storage when the btree is created (or early on)?  Otherwise, every time we need the bucket_storage I think we need to:  look at btree->dhandle->type, if it is TIERED, then cast the dhandle to a ?? to get the bucket storage.  Somehow we need to get to the WT_TIERED to get the WT_BUCKET_STORAGE, I don't see how to do that.